application/api/spec/api-exported.yaml

```diff
@@ -846,6 +846,18 @@ paths:
                 oneOf:
                   - $ref: '#/components/schemas/definitions-responses-Status-yaml'
                   - $ref: '#/components/schemas/response-unsignedRequestError'
+  '/admin-api/rpc#schema.get':
```

Comment:
Did you make PR to spec repo for this change?

Reply:
nope